src/stages/stage6/spells/spells.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * This software is licensed under the terms of the MIT License.
+ * See COPYING for further information.
+ * ---
+ * Copyright (c) 2011-2019, Lukas Weber <laochailan@web.de>.
+ * Copyright (c) 2012-2019, Andrei Alexeyev <akari@taisei-project.org>.
+*/
+
+#ifndef IGUARD_stages_stage6_spells_spells_h
+#define IGUARD_stages_stage6_spells_spells_h
+
+#include "stages/stage6/elly.h"
+#include "taisei.h"
```

Comment:
`taisei.h` must be the first include in all files. Also, don't write/copy-paste the license/include-guard/etc. boilerplate manually. After you add or move/rename a file, simply run `ninja upkeep`. It'll generate them for you.